doc/sklearn.rst

```diff
@@ -52,6 +54,40 @@ on all added and monkey-patched classes, which are discovered by means of
 introspection. This assures scikit-learn API compatibility of all
 `daal4py.sklearn` classes.
 
+.. _sklearn_verbose:
+
+scikit-learn verbose
+----------------
+
+In order to find out which implementation of the algorithm is currently used,
+export the environment variable.
+
+On Linux and Mac OS::
+
+    export IDP_SKLEARN_VERBOSE=INFO
+
+On Windows::
+
+    set IDP_SKLEARN_VERBOSE=INFO
+
+For example, on Linux you may use the following command::
```

Comment:
what's that an example of exactly? how's is that different from the command above? I think you need to add a bit more context here (e.g. something like "An example below shows how to ... ")

Reply:
I agree, this example duplicates command above. I removed this.